docs/stdlib/generic.rst

```diff
@@ -235,12 +245,10 @@ Generic
 
     :index: length count array
 
-    A polymorphic function to calculate a "length" of its first
-    argument.
+    Returns an :eql:type:`int64` of counted values within a type.
 
-    Return the number of characters in a :eql:type:`str`, or the
-    number of bytes in :eql:type:`bytes`, or the number of elements in
-    an :eql:type:`array`.
+    This function works with the :eql:type:`str`, :eql:type:`bytes` and
+    :eql:type:`array` types. Strings and bytes will return a scalar value:
```

Comment:
It will return a scalar value for all types, right? If you wanted to bring the return type out of the signature and repeat it in another location, you could replace that sentence with something like "It will return the count as an int64," and I feel like that would be more specific and more useful. I would lean toward omitting it altogether.

```suggestion
    :eql:type:`array` types:
```